### ISSUE TITLE
Not working on firefox and github

### ISSUE THREAD

**Issue by matheusfillipe** [glacambre/firenvim#1046]
Is not working on github with firefox anymore. Seems the ctrl+E shortcut for markdown is overwriting firenvim's. Also I don't see on the extensions a preferences page to change the default shortcut anymore.

- OS Version: linux x86-64
- Browser Version:  Firefox 88.0
- Browser Addon Version: 0.2.6
- Neovim Plugin Version: 0.2.6
- What I tried to do:
- What happened:

**Comment by glacambre** [comment]
>I don't see on the extensions a preferences page to change the default shortcut anymore.

Go to `about:addons`, click on the cog icon in the top right corner, select `Manage Extension Shortcuts`.

**Comment by matheusfillipe** [comment]
Oh ok that is not missing, I changed and it woks. Still the github shortcuts are overwriting it which is still a minor issue I guess?

**Comment by glacambre** [comment]
@matheusfillipe Does the keyboard shortcut start working if you set `Shift+Alt+A` as keyboard shortcut?

**Comment by glacambre** [comment]
That's a firefox bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1713794

**Comment by matheusfillipe** [comment]
Yes it does. As I said i set it to `Alt+Space` and it was already working. It just doesn't with `Ctrl+E`. Well if it is a firefox bug I guess this can be closed.

**Comment by glacambre** [comment]
>As I said i set it to Alt+Space

I'm not seeing that anywhere in the ticket :)

>if it is a firefox bug I guess this can be closed.

I'll keep it open, otherwise other people will probably open new issues. Thanks for the report :)

**Comment by matheusfillipe** [comment]
Oh sorry, I really thought I mentioned about changing the shortcut to `Alt+Space`. No problem!

**Comment by glacambre** [comment]
Firefox now lets you decide whether keyboard shortcuts belong to the browser or to the extension with `about:keyboard`, so this issue can be closed.
